Submitted By: Jeremy Huntwork (jhuntwork at linuxfromscratch dot org)
Date: 2005-10-20
Initial Package Version: 2.6.12.0
Origin: Created for Cross-LFS
Upstream Status: Not submitted - LFS Specific
Description: Creates kernel header stubs for biarch builds, specific to ppc arch.
             Based on cross-lfs scripts by Ryan Oliver

diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/a.out.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/a.out.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/a.out.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/a.out.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__A_OUT_H__
+#define __STUB__A_OUT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/a.out.h>
+#else
+#include <asm-ppc/a.out.h>
+#endif
+
+#endif /* __STUB__A_OUT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/abs_addr.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/abs_addr.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/abs_addr.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/abs_addr.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ABS_ADDR_H__
+#define __STUB__ABS_ADDR_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/abs_addr.h>
+#endif
+
+#endif /* __STUB__ABS_ADDR_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/agp.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/agp.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/agp.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/agp.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__AGP_H__
+#define __STUB__AGP_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/agp.h>
+#else
+#include <asm-ppc/agp.h>
+#endif
+
+#endif /* __STUB__AGP_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/amigayle.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/amigayle.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/amigayle.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/amigayle.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__AMIGAYLE_H__
+#define __STUB__AMIGAYLE_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/amigayle.h>
+#endif
+
+#endif /* __STUB__AMIGAYLE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/amipcmcia.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/amipcmcia.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/amipcmcia.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/amipcmcia.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__AMIPCMCIA_H__
+#define __STUB__AMIPCMCIA_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/amipcmcia.h>
+#endif
+
+#endif /* __STUB__AMIPCMCIA_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ans-lcd.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ans-lcd.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ans-lcd.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ans-lcd.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ANS-LCD_H__
+#define __STUB__ANS-LCD_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/ans-lcd.h>
+#endif
+
+#endif /* __STUB__ANS-LCD_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/atomic.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/atomic.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/atomic.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/atomic.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__ATOMIC_H__
+#define __STUB__ATOMIC_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/atomic.h>
+#else
+#include <asm-ppc/atomic.h>
+#endif
+
+#endif /* __STUB__ATOMIC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/bitops.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/bitops.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/bitops.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/bitops.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__BITOPS_H__
+#define __STUB__BITOPS_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/bitops.h>
+#else
+#include <asm-ppc/bitops.h>
+#endif
+
+#endif /* __STUB__BITOPS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/bootx.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/bootx.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/bootx.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/bootx.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__BOOTX_H__
+#define __STUB__BOOTX_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/bootx.h>
+#endif
+
+#endif /* __STUB__BOOTX_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/bseip.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/bseip.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/bseip.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/bseip.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__BSEIP_H__
+#define __STUB__BSEIP_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/bseip.h>
+#endif
+
+#endif /* __STUB__BSEIP_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/byteorder.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/byteorder.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/byteorder.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/byteorder.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__BYTEORDER_H__
+#define __STUB__BYTEORDER_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/byteorder.h>
+#else
+#include <asm-ppc/byteorder.h>
+#endif
+
+#endif /* __STUB__BYTEORDER_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/cache.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/cache.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/cache.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/cache.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__CACHE_H__
+#define __STUB__CACHE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/cache.h>
+#else
+#include <asm-ppc/cache.h>
+#endif
+
+#endif /* __STUB__CACHE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/cacheflush.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/cacheflush.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/cacheflush.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/cacheflush.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__CACHEFLUSH_H__
+#define __STUB__CACHEFLUSH_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/cacheflush.h>
+#else
+#include <asm-ppc/cacheflush.h>
+#endif
+
+#endif /* __STUB__CACHEFLUSH_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/checksum.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/checksum.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/checksum.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/checksum.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__CHECKSUM_H__
+#define __STUB__CHECKSUM_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/checksum.h>
+#else
+#include <asm-ppc/checksum.h>
+#endif
+
+#endif /* __STUB__CHECKSUM_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/commproc.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/commproc.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/commproc.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/commproc.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__COMMPROC_H__
+#define __STUB__COMMPROC_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/commproc.h>
+#endif
+
+#endif /* __STUB__COMMPROC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/compat.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/compat.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/compat.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/compat.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__COMPAT_H__
+#define __STUB__COMPAT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/compat.h>
+#endif
+
+#endif /* __STUB__COMPAT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/cputable.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/cputable.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/cputable.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/cputable.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__CPUTABLE_H__
+#define __STUB__CPUTABLE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/cputable.h>
+#else
+#include <asm-ppc/cputable.h>
+#endif
+
+#endif /* __STUB__CPUTABLE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/dma.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/dma.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/dma.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/dma.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__DMA_H__
+#define __STUB__DMA_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/dma.h>
+#else
+#include <asm-ppc/dma.h>
+#endif
+
+#endif /* __STUB__DMA_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/eeh.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/eeh.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/eeh.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/eeh.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__EEH_H__
+#define __STUB__EEH_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/eeh.h>
+#endif
+
+#endif /* __STUB__EEH_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/elf.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/elf.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/elf.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/elf.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__ELF_H__
+#define __STUB__ELF_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/elf.h>
+#else
+#include <asm-ppc/elf.h>
+#endif
+
+#endif /* __STUB__ELF_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/errno.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/errno.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/errno.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/errno.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__ERRNO_H__
+#define __STUB__ERRNO_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/errno.h>
+#else
+#include <asm-ppc/errno.h>
+#endif
+
+#endif /* __STUB__ERRNO_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/fcntl.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/fcntl.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/fcntl.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/fcntl.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__FCNTL_H__
+#define __STUB__FCNTL_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/fcntl.h>
+#else
+#include <asm-ppc/fcntl.h>
+#endif
+
+#endif /* __STUB__FCNTL_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/floppy.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/floppy.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/floppy.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/floppy.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__FLOPPY_H__
+#define __STUB__FLOPPY_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/floppy.h>
+#else
+#include <asm-ppc/floppy.h>
+#endif
+
+#endif /* __STUB__FLOPPY_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/gg2.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/gg2.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/gg2.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/gg2.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__GG2_H__
+#define __STUB__GG2_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/gg2.h>
+#endif
+
+#endif /* __STUB__GG2_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/gt64260.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/gt64260.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/gt64260.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/gt64260.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__GT64260_H__
+#define __STUB__GT64260_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/gt64260.h>
+#endif
+
+#endif /* __STUB__GT64260_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/gt64260_defs.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/gt64260_defs.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/gt64260_defs.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/gt64260_defs.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__GT64260_DEFS_H__
+#define __STUB__GT64260_DEFS_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/gt64260_defs.h>
+#endif
+
+#endif /* __STUB__GT64260_DEFS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/hardirq.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/hardirq.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/hardirq.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/hardirq.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__HARDIRQ_H__
+#define __STUB__HARDIRQ_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/hardirq.h>
+#else
+#include <asm-ppc/hardirq.h>
+#endif
+
+#endif /* __STUB__HARDIRQ_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/harrier.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/harrier.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/harrier.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/harrier.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HARRIER_H__
+#define __STUB__HARRIER_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/harrier.h>
+#endif
+
+#endif /* __STUB__HARRIER_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/hdreg.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/hdreg.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/hdreg.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/hdreg.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__HDREG_H__
+#define __STUB__HDREG_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/hdreg.h>
+#else
+#include <asm-ppc/hdreg.h>
+#endif
+
+#endif /* __STUB__HDREG_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/heathrow.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/heathrow.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/heathrow.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/heathrow.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HEATHROW_H__
+#define __STUB__HEATHROW_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/heathrow.h>
+#endif
+
+#endif /* __STUB__HEATHROW_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/hvcall.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/hvcall.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/hvcall.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/hvcall.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVCALL_H__
+#define __STUB__HVCALL_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/hvcall.h>
+#endif
+
+#endif /* __STUB__HVCALL_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/hw_irq.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/hw_irq.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/hw_irq.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/hw_irq.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__HW_IRQ_H__
+#define __STUB__HW_IRQ_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/hw_irq.h>
+#else
+#include <asm-ppc/hw_irq.h>
+#endif
+
+#endif /* __STUB__HW_IRQ_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/i8259.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/i8259.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/i8259.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/i8259.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__I8259_H__
+#define __STUB__I8259_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/i8259.h>
+#endif
+
+#endif /* __STUB__I8259_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCall.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCall.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCall.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCall.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVCALL_H__
+#define __STUB__HVCALL_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvCall.h>
+#endif
+
+#endif /* __STUB__HVCALL_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallCfg.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallCfg.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallCfg.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallCfg.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVCALLCFG_H__
+#define __STUB__HVCALLCFG_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvCallCfg.h>
+#endif
+
+#endif /* __STUB__HVCALLCFG_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallEvent.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallEvent.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallEvent.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallEvent.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVCALLEVENT_H__
+#define __STUB__HVCALLEVENT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvCallEvent.h>
+#endif
+
+#endif /* __STUB__HVCALLEVENT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallHpt.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallHpt.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallHpt.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallHpt.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVCALLHPT_H__
+#define __STUB__HVCALLHPT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvCallHpt.h>
+#endif
+
+#endif /* __STUB__HVCALLHPT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallPci.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallPci.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallPci.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallPci.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVCALLPCI_H__
+#define __STUB__HVCALLPCI_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvCallPci.h>
+#endif
+
+#endif /* __STUB__HVCALLPCI_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallSc.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallSc.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallSc.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallSc.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVCALLSC_H__
+#define __STUB__HVCALLSC_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvCallSc.h>
+#endif
+
+#endif /* __STUB__HVCALLSC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallSm.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallSm.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallSm.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallSm.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVCALLSM_H__
+#define __STUB__HVCALLSM_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvCallSm.h>
+#endif
+
+#endif /* __STUB__HVCALLSM_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallXm.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallXm.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvCallXm.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvCallXm.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVCALLXM_H__
+#define __STUB__HVCALLXM_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvCallXm.h>
+#endif
+
+#endif /* __STUB__HVCALLXM_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvLpConfig.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvLpConfig.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvLpConfig.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvLpConfig.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVLPCONFIG_H__
+#define __STUB__HVLPCONFIG_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvLpConfig.h>
+#endif
+
+#endif /* __STUB__HVLPCONFIG_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvLpEvent.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvLpEvent.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvLpEvent.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvLpEvent.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVLPEVENT_H__
+#define __STUB__HVLPEVENT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvLpEvent.h>
+#endif
+
+#endif /* __STUB__HVLPEVENT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvReleaseData.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvReleaseData.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvReleaseData.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvReleaseData.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVRELEASEDATA_H__
+#define __STUB__HVRELEASEDATA_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvReleaseData.h>
+#endif
+
+#endif /* __STUB__HVRELEASEDATA_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvTypes.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvTypes.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/HvTypes.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/HvTypes.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__HVTYPES_H__
+#define __STUB__HVTYPES_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/HvTypes.h>
+#endif
+
+#endif /* __STUB__HVTYPES_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/IoHriMainStore.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/IoHriMainStore.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/IoHriMainStore.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/IoHriMainStore.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__IOHRIMAINSTORE_H__
+#define __STUB__IOHRIMAINSTORE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/IoHriMainStore.h>
+#endif
+
+#endif /* __STUB__IOHRIMAINSTORE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/IoHriProcessorVpd.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/IoHriProcessorVpd.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/IoHriProcessorVpd.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/IoHriProcessorVpd.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__IOHRIPROCESSORVPD_H__
+#define __STUB__IOHRIPROCESSORVPD_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/IoHriProcessorVpd.h>
+#endif
+
+#endif /* __STUB__IOHRIPROCESSORVPD_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItExtVpdPanel.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItExtVpdPanel.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItExtVpdPanel.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItExtVpdPanel.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ITEXTVPDPANEL_H__
+#define __STUB__ITEXTVPDPANEL_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/ItExtVpdPanel.h>
+#endif
+
+#endif /* __STUB__ITEXTVPDPANEL_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItIplParmsReal.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItIplParmsReal.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItIplParmsReal.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItIplParmsReal.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ITIPLPARMSREAL_H__
+#define __STUB__ITIPLPARMSREAL_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/ItIplParmsReal.h>
+#endif
+
+#endif /* __STUB__ITIPLPARMSREAL_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItLpNaca.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItLpNaca.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItLpNaca.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItLpNaca.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ITLPNACA_H__
+#define __STUB__ITLPNACA_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/ItLpNaca.h>
+#endif
+
+#endif /* __STUB__ITLPNACA_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItLpQueue.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItLpQueue.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItLpQueue.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItLpQueue.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ITLPQUEUE_H__
+#define __STUB__ITLPQUEUE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/ItLpQueue.h>
+#endif
+
+#endif /* __STUB__ITLPQUEUE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItLpRegSave.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItLpRegSave.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItLpRegSave.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItLpRegSave.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ITLPREGSAVE_H__
+#define __STUB__ITLPREGSAVE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/ItLpRegSave.h>
+#endif
+
+#endif /* __STUB__ITLPREGSAVE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItSpCommArea.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItSpCommArea.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItSpCommArea.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItSpCommArea.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ITSPCOMMAREA_H__
+#define __STUB__ITSPCOMMAREA_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/ItSpCommArea.h>
+#endif
+
+#endif /* __STUB__ITSPCOMMAREA_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItVpdAreas.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItVpdAreas.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/ItVpdAreas.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/ItVpdAreas.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ITVPDAREAS_H__
+#define __STUB__ITVPDAREAS_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/ItVpdAreas.h>
+#endif
+
+#endif /* __STUB__ITVPDAREAS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/LparData.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/LparData.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/LparData.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/LparData.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__LPARDATA_H__
+#define __STUB__LPARDATA_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/LparData.h>
+#endif
+
+#endif /* __STUB__LPARDATA_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/LparMap.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/LparMap.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/LparMap.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/LparMap.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__LPARMAP_H__
+#define __STUB__LPARMAP_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/LparMap.h>
+#endif
+
+#endif /* __STUB__LPARMAP_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/XmPciLpEvent.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/XmPciLpEvent.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/XmPciLpEvent.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/XmPciLpEvent.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__XMPCILPEVENT_H__
+#define __STUB__XMPCILPEVENT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/XmPciLpEvent.h>
+#endif
+
+#endif /* __STUB__XMPCILPEVENT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/iSeries_io.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/iSeries_io.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/iSeries_io.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/iSeries_io.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ISERIES_IO_H__
+#define __STUB__ISERIES_IO_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/iSeries_io.h>
+#endif
+
+#endif /* __STUB__ISERIES_IO_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/iSeries_irq.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/iSeries_irq.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/iSeries_irq.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/iSeries_irq.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ISERIES_IRQ_H__
+#define __STUB__ISERIES_IRQ_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/iSeries_irq.h>
+#endif
+
+#endif /* __STUB__ISERIES_IRQ_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/iSeries_pci.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/iSeries_pci.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/iSeries_pci.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/iSeries_pci.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ISERIES_PCI_H__
+#define __STUB__ISERIES_PCI_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/iSeries_pci.h>
+#endif
+
+#endif /* __STUB__ISERIES_PCI_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/iSeries_proc.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/iSeries_proc.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/iSeries_proc.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/iSeries_proc.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__ISERIES_PROC_H__
+#define __STUB__ISERIES_PROC_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/iSeries_proc.h>
+#endif
+
+#endif /* __STUB__ISERIES_PROC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/mf.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/mf.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/iSeries/mf.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/iSeries/mf.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__MF_H__
+#define __STUB__MF_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/iSeries/mf.h>
+#endif
+
+#endif /* __STUB__MF_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/imalloc.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/imalloc.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/imalloc.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/imalloc.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__IMALLOC_H__
+#define __STUB__IMALLOC_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/imalloc.h>
+#endif
+
+#endif /* __STUB__IMALLOC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/init.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/init.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/init.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/init.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__INIT_H__
+#define __STUB__INIT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/init.h>
+#endif
+
+#endif /* __STUB__INIT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/io.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/io.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/io.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/io.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__IO_H__
+#define __STUB__IO_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/io.h>
+#else
+#include <asm-ppc/io.h>
+#endif
+
+#endif /* __STUB__IO_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ioctl.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ioctl.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ioctl.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ioctl.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__IOCTL_H__
+#define __STUB__IOCTL_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ioctl.h>
+#else
+#include <asm-ppc/ioctl.h>
+#endif
+
+#endif /* __STUB__IOCTL_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ioctls.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ioctls.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ioctls.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ioctls.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__IOCTLS_H__
+#define __STUB__IOCTLS_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ioctls.h>
+#else
+#include <asm-ppc/ioctls.h>
+#endif
+
+#endif /* __STUB__IOCTLS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ipc.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ipc.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ipc.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ipc.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__IPC_H__
+#define __STUB__IPC_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ipc.h>
+#else
+#include <asm-ppc/ipc.h>
+#endif
+
+#endif /* __STUB__IPC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ipcbuf.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ipcbuf.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ipcbuf.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ipcbuf.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__IPCBUF_H__
+#define __STUB__IPCBUF_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ipcbuf.h>
+#else
+#include <asm-ppc/ipcbuf.h>
+#endif
+
+#endif /* __STUB__IPCBUF_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/irq.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/irq.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/irq.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/irq.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__IRQ_H__
+#define __STUB__IRQ_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/irq.h>
+#else
+#include <asm-ppc/irq.h>
+#endif
+
+#endif /* __STUB__IRQ_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/keylargo.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/keylargo.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/keylargo.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/keylargo.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__KEYLARGO_H__
+#define __STUB__KEYLARGO_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/keylargo.h>
+#endif
+
+#endif /* __STUB__KEYLARGO_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/linkage.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/linkage.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/linkage.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/linkage.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__LINKAGE_H__
+#define __STUB__LINKAGE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/linkage.h>
+#else
+#include <asm-ppc/linkage.h>
+#endif
+
+#endif /* __STUB__LINKAGE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/lmb.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/lmb.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/lmb.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/lmb.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__LMB_H__
+#define __STUB__LMB_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/lmb.h>
+#endif
+
+#endif /* __STUB__LMB_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/m48t35.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/m48t35.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/m48t35.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/m48t35.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__M48T35_H__
+#define __STUB__M48T35_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/m48t35.h>
+#endif
+
+#endif /* __STUB__M48T35_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/machdep.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/machdep.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/machdep.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/machdep.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__MACHDEP_H__
+#define __STUB__MACHDEP_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/machdep.h>
+#else
+#include <asm-ppc/machdep.h>
+#endif
+
+#endif /* __STUB__MACHDEP_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/macio.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/macio.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/macio.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/macio.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__MACIO_H__
+#define __STUB__MACIO_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/macio.h>
+#endif
+
+#endif /* __STUB__MACIO_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/memory.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/memory.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/memory.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/memory.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__MEMORY_H__
+#define __STUB__MEMORY_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/memory.h>
+#endif
+
+#endif /* __STUB__MEMORY_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/mk48t59.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/mk48t59.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/mk48t59.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/mk48t59.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__MK48T59_H__
+#define __STUB__MK48T59_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/mk48t59.h>
+#endif
+
+#endif /* __STUB__MK48T59_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/mman.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/mman.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/mman.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/mman.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__MMAN_H__
+#define __STUB__MMAN_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/mman.h>
+#else
+#include <asm-ppc/mman.h>
+#endif
+
+#endif /* __STUB__MMAN_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/mmu.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/mmu.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/mmu.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/mmu.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__MMU_H__
+#define __STUB__MMU_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/mmu.h>
+#else
+#include <asm-ppc/mmu.h>
+#endif
+
+#endif /* __STUB__MMU_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/mmu_context.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/mmu_context.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/mmu_context.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/mmu_context.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__MMU_CONTEXT_H__
+#define __STUB__MMU_CONTEXT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/mmu_context.h>
+#else
+#include <asm-ppc/mmu_context.h>
+#endif
+
+#endif /* __STUB__MMU_CONTEXT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/module.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/module.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/module.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/module.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__MODULE_H__
+#define __STUB__MODULE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/module.h>
+#else
+#include <asm-ppc/module.h>
+#endif
+
+#endif /* __STUB__MODULE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/msgbuf.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/msgbuf.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/msgbuf.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/msgbuf.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__MSGBUF_H__
+#define __STUB__MSGBUF_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/msgbuf.h>
+#else
+#include <asm-ppc/msgbuf.h>
+#endif
+
+#endif /* __STUB__MSGBUF_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/naca.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/naca.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/naca.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/naca.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__NACA_H__
+#define __STUB__NACA_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/naca.h>
+#endif
+
+#endif /* __STUB__NACA_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/namei.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/namei.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/namei.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/namei.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__NAMEI_H__
+#define __STUB__NAMEI_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/namei.h>
+#else
+#include <asm-ppc/namei.h>
+#endif
+
+#endif /* __STUB__NAMEI_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/numnodes.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/numnodes.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/numnodes.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/numnodes.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__NUMNODES_H__
+#define __STUB__NUMNODES_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/numnodes.h>
+#endif
+
+#endif /* __STUB__NUMNODES_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/nvram.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/nvram.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/nvram.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/nvram.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__NVRAM_H__
+#define __STUB__NVRAM_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/nvram.h>
+#else
+#include <asm-ppc/nvram.h>
+#endif
+
+#endif /* __STUB__NVRAM_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ocp_ids.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ocp_ids.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ocp_ids.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ocp_ids.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__OCP_IDS_H__
+#define __STUB__OCP_IDS_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/ocp_ids.h>
+#endif
+
+#endif /* __STUB__OCP_IDS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/of_device.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/of_device.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/of_device.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/of_device.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__OF_DEVICE_H__
+#define __STUB__OF_DEVICE_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/of_device.h>
+#endif
+
+#endif /* __STUB__OF_DEVICE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ohare.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ohare.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ohare.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ohare.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__OHARE_H__
+#define __STUB__OHARE_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/ohare.h>
+#endif
+
+#endif /* __STUB__OHARE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/open_pic.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/open_pic.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/open_pic.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/open_pic.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__OPEN_PIC_H__
+#define __STUB__OPEN_PIC_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/open_pic.h>
+#endif
+
+#endif /* __STUB__OPEN_PIC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/pSeries_reconfig.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/pSeries_reconfig.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/pSeries_reconfig.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/pSeries_reconfig.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__PSERIES_RECONFIG_H__
+#define __STUB__PSERIES_RECONFIG_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/pSeries_reconfig.h>
+#endif
+
+#endif /* __STUB__PSERIES_RECONFIG_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/paca.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/paca.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/paca.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/paca.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__PACA_H__
+#define __STUB__PACA_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/paca.h>
+#endif
+
+#endif /* __STUB__PACA_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/page.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/page.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/page.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/page.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__PAGE_H__
+#define __STUB__PAGE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/page.h>
+#else
+#include <asm-ppc/page.h>
+#endif
+
+#endif /* __STUB__PAGE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/param.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/param.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/param.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/param.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__PARAM_H__
+#define __STUB__PARAM_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/param.h>
+#else
+#include <asm-ppc/param.h>
+#endif
+
+#endif /* __STUB__PARAM_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/parport.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/parport.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/parport.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/parport.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__PARPORT_H__
+#define __STUB__PARPORT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/parport.h>
+#else
+#include <asm-ppc/parport.h>
+#endif
+
+#endif /* __STUB__PARPORT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/pgalloc.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/pgalloc.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/pgalloc.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/pgalloc.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__PGALLOC_H__
+#define __STUB__PGALLOC_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/pgalloc.h>
+#else
+#include <asm-ppc/pgalloc.h>
+#endif
+
+#endif /* __STUB__PGALLOC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/pmc.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/pmc.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/pmc.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/pmc.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__PMC_H__
+#define __STUB__PMC_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/pmc.h>
+#endif
+
+#endif /* __STUB__PMC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/poll.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/poll.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/poll.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/poll.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__POLL_H__
+#define __STUB__POLL_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/poll.h>
+#else
+#include <asm-ppc/poll.h>
+#endif
+
+#endif /* __STUB__POLL_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/posix_types.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/posix_types.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/posix_types.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/posix_types.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__POSIX_TYPES_H__
+#define __STUB__POSIX_TYPES_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/posix_types.h>
+#else
+#include <asm-ppc/posix_types.h>
+#endif
+
+#endif /* __STUB__POSIX_TYPES_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ppc32.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ppc32.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ppc32.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ppc32.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__PPC32_H__
+#define __STUB__PPC32_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ppc32.h>
+#endif
+
+#endif /* __STUB__PPC32_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ppc_asm.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ppc_asm.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ppc_asm.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ppc_asm.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__PPC_ASM_H__
+#define __STUB__PPC_ASM_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ppc_asm.h>
+#else
+#include <asm-ppc/ppc_asm.h>
+#endif
+
+#endif /* __STUB__PPC_ASM_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ppcboot.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ppcboot.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ppcboot.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ppcboot.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__PPCBOOT_H__
+#define __STUB__PPCBOOT_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/ppcboot.h>
+#endif
+
+#endif /* __STUB__PPCBOOT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ppcdebug.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ppcdebug.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ppcdebug.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ppcdebug.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__PPCDEBUG_H__
+#define __STUB__PPCDEBUG_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ppcdebug.h>
+#endif
+
+#endif /* __STUB__PPCDEBUG_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/processor.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/processor.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/processor.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/processor.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__PROCESSOR_H__
+#define __STUB__PROCESSOR_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/processor.h>
+#else
+#include <asm-ppc/processor.h>
+#endif
+
+#endif /* __STUB__PROCESSOR_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ptrace-common.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ptrace-common.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ptrace-common.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ptrace-common.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__PTRACE-COMMON_H__
+#define __STUB__PTRACE-COMMON_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ptrace-common.h>
+#endif
+
+#endif /* __STUB__PTRACE-COMMON_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ptrace.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ptrace.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ptrace.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ptrace.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__PTRACE_H__
+#define __STUB__PTRACE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ptrace.h>
+#else
+#include <asm-ppc/ptrace.h>
+#endif
+
+#endif /* __STUB__PTRACE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/reg.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/reg.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/reg.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/reg.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__REG_H__
+#define __STUB__REG_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/reg.h>
+#endif
+
+#endif /* __STUB__REG_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/rtas.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/rtas.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/rtas.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/rtas.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__RTAS_H__
+#define __STUB__RTAS_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/rtas.h>
+#endif
+
+#endif /* __STUB__RTAS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/segment.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/segment.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/segment.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/segment.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SEGMENT_H__
+#define __STUB__SEGMENT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/segment.h>
+#else
+#include <asm-ppc/segment.h>
+#endif
+
+#endif /* __STUB__SEGMENT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/semaphore.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/semaphore.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/semaphore.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/semaphore.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SEMAPHORE_H__
+#define __STUB__SEMAPHORE_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/semaphore.h>
+#else
+#include <asm-ppc/semaphore.h>
+#endif
+
+#endif /* __STUB__SEMAPHORE_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/sembuf.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/sembuf.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/sembuf.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/sembuf.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SEMBUF_H__
+#define __STUB__SEMBUF_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/sembuf.h>
+#else
+#include <asm-ppc/sembuf.h>
+#endif
+
+#endif /* __STUB__SEMBUF_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/serial.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/serial.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/serial.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/serial.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__SERIAL_H__
+#define __STUB__SERIAL_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/serial.h>
+#endif
+
+#endif /* __STUB__SERIAL_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/shmbuf.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/shmbuf.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/shmbuf.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/shmbuf.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SHMBUF_H__
+#define __STUB__SHMBUF_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/shmbuf.h>
+#else
+#include <asm-ppc/shmbuf.h>
+#endif
+
+#endif /* __STUB__SHMBUF_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/shmparam.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/shmparam.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/shmparam.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/shmparam.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SHMPARAM_H__
+#define __STUB__SHMPARAM_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/shmparam.h>
+#else
+#include <asm-ppc/shmparam.h>
+#endif
+
+#endif /* __STUB__SHMPARAM_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/sigcontext.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/sigcontext.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/sigcontext.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/sigcontext.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SIGCONTEXT_H__
+#define __STUB__SIGCONTEXT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/sigcontext.h>
+#else
+#include <asm-ppc/sigcontext.h>
+#endif
+
+#endif /* __STUB__SIGCONTEXT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/siginfo.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/siginfo.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/siginfo.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/siginfo.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SIGINFO_H__
+#define __STUB__SIGINFO_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/siginfo.h>
+#else
+#include <asm-ppc/siginfo.h>
+#endif
+
+#endif /* __STUB__SIGINFO_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/smu.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/smu.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/smu.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/smu.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__SMU_H__
+#define __STUB__SMU_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/smu.h>
+#endif
+
+#endif /* __STUB__SMU_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/socket.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/socket.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/socket.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/socket.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SOCKET_H__
+#define __STUB__SOCKET_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/socket.h>
+#else
+#include <asm-ppc/socket.h>
+#endif
+
+#endif /* __STUB__SOCKET_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/sockios.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/sockios.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/sockios.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/sockios.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SOCKIOS_H__
+#define __STUB__SOCKIOS_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/sockios.h>
+#else
+#include <asm-ppc/sockios.h>
+#endif
+
+#endif /* __STUB__SOCKIOS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/spinlock.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/spinlock.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/spinlock.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/spinlock.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SPINLOCK_H__
+#define __STUB__SPINLOCK_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/spinlock.h>
+#else
+#include <asm-ppc/spinlock.h>
+#endif
+
+#endif /* __STUB__SPINLOCK_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/stat.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/stat.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/stat.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/stat.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__STAT_H__
+#define __STUB__STAT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/stat.h>
+#else
+#include <asm-ppc/stat.h>
+#endif
+
+#endif /* __STUB__STAT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/suspend.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/suspend.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/suspend.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/suspend.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__SUSPEND_H__
+#define __STUB__SUSPEND_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/suspend.h>
+#endif
+
+#endif /* __STUB__SUSPEND_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/system.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/system.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/system.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/system.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__SYSTEM_H__
+#define __STUB__SYSTEM_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/system.h>
+#else
+#include <asm-ppc/system.h>
+#endif
+
+#endif /* __STUB__SYSTEM_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/systemcfg.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/systemcfg.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/systemcfg.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/systemcfg.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__SYSTEMCFG_H__
+#define __STUB__SYSTEMCFG_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/systemcfg.h>
+#endif
+
+#endif /* __STUB__SYSTEMCFG_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/termbits.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/termbits.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/termbits.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/termbits.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__TERMBITS_H__
+#define __STUB__TERMBITS_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/termbits.h>
+#else
+#include <asm-ppc/termbits.h>
+#endif
+
+#endif /* __STUB__TERMBITS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/termios.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/termios.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/termios.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/termios.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__TERMIOS_H__
+#define __STUB__TERMIOS_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/termios.h>
+#else
+#include <asm-ppc/termios.h>
+#endif
+
+#endif /* __STUB__TERMIOS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/timex.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/timex.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/timex.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/timex.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__TIMEX_H__
+#define __STUB__TIMEX_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/timex.h>
+#else
+#include <asm-ppc/timex.h>
+#endif
+
+#endif /* __STUB__TIMEX_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/tlbflush.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/tlbflush.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/tlbflush.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/tlbflush.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__TLBFLUSH_H__
+#define __STUB__TLBFLUSH_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/tlbflush.h>
+#else
+#include <asm-ppc/tlbflush.h>
+#endif
+
+#endif /* __STUB__TLBFLUSH_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/todc.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/todc.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/todc.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/todc.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__TODC_H__
+#define __STUB__TODC_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/todc.h>
+#endif
+
+#endif /* __STUB__TODC_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/traps.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/traps.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/traps.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/traps.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__TRAPS_H__
+#define __STUB__TRAPS_H__
+
+#ifndef __powerpc64__
+#include <asm-ppc/traps.h>
+#endif
+
+#endif /* __STUB__TRAPS_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/types.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/types.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/types.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/types.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__TYPES_H__
+#define __STUB__TYPES_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/types.h>
+#else
+#include <asm-ppc/types.h>
+#endif
+
+#endif /* __STUB__TYPES_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ucontext.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ucontext.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/ucontext.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/ucontext.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__UCONTEXT_H__
+#define __STUB__UCONTEXT_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/ucontext.h>
+#else
+#include <asm-ppc/ucontext.h>
+#endif
+
+#endif /* __STUB__UCONTEXT_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/udbg.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/udbg.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/udbg.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/udbg.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__UDBG_H__
+#define __STUB__UDBG_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/udbg.h>
+#endif
+
+#endif /* __STUB__UDBG_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/unaligned.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/unaligned.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/unaligned.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/unaligned.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__UNALIGNED_H__
+#define __STUB__UNALIGNED_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/unaligned.h>
+#else
+#include <asm-ppc/unaligned.h>
+#endif
+
+#endif /* __STUB__UNALIGNED_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/unistd.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/unistd.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/unistd.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/unistd.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__UNISTD_H__
+#define __STUB__UNISTD_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/unistd.h>
+#else
+#include <asm-ppc/unistd.h>
+#endif
+
+#endif /* __STUB__UNISTD_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/user.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/user.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/user.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/user.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__USER_H__
+#define __STUB__USER_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/user.h>
+#else
+#include <asm-ppc/user.h>
+#endif
+
+#endif /* __STUB__USER_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/vga.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/vga.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/vga.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/vga.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,10 @@
+#ifndef __STUB__VGA_H__
+#define __STUB__VGA_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/vga.h>
+#else
+#include <asm-ppc/vga.h>
+#endif
+
+#endif /* __STUB__VGA_H__ */
diff -Naur linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/xics.h linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/xics.h
--- linux-libc-headers-2.6.12.0.orig/include/asm-ppc-biarch/xics.h	Wed Dec 31 19:00:00 1969
+++ linux-libc-headers-2.6.12.0/include/asm-ppc-biarch/xics.h	Thu Oct 20 12:39:51 2005
@@ -0,0 +1,8 @@
+#ifndef __STUB__XICS_H__
+#define __STUB__XICS_H__
+
+#ifdef __powerpc64__
+#include <asm-ppc64/xics.h>
+#endif
+
+#endif /* __STUB__XICS_H__ */
